app/helpers/users_helper.rb

```diff
@@ -2,8 +2,4 @@ module UsersHelper
   def user_search_filtered?
```

Comment:
We can remove this method ... and the file. It's not being used anymore.

Reply:
Double check with the authors @CZagrobelny ? I don't know what the norms of this project are so sorry if you already have access to that insight @latazzajones !